### PR TITLE
chromium: refresh

### PR DESCRIPTION
**Pull request by navarroaxel**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 98.0.4758.102

### CODE REVIEW

File: pages/common/chromium.md [hunk 1]
```diff
@@ -21,8 +17,20 @@
...
+
+- Open with a custom profile directory:
+
+`chromium --user-data-dir={{path/to/directory}}`
+
+- Open without CORS validation, useful to test an API:
+
+`chromium --user-data-dir={{path/to/directory}} --disable-web-security`
+
+- Open and open a DevTools window for each tab opened:
```
Comment (EmilySeville7cfg) [inline pages/common/chromium.md:34]:
```suggestion
- Open with a DevTools window for each tab opened:
```

File: pages/common/chromium.md [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
 
-- Open a file:
+- Open a specific URL or file:
 
-`chromium {{path/to/file.html}}`
-
-- Open a URL:
-
-`chromium {{example.com}}`
+`chromium {{url|path/to/file.html}}`
```
Comment (EmilySeville7cfg) [inline pages/common/chromium.md:8]:
```suggestion
`chromium {{https://example.com|path/to/file.html}}`
```

Maybe this is better but more verbose.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/chromium.md [hunk 3]
```diff
@@ -1,15 +1,11 @@
 # chromium
 
-> Open-source web browser from Google.
+> Open-source web browser principally developed and maintained by Google.
```
Comment (mfrw) [inline pages/common/chromium.md:3]: +1 :)
this reads much better

File: pages/common/chromium.md [hunk 4]
```diff
@@ -21,8 +17,20 @@
...
 
 - Use a proxy server:
 
 `chromium --proxy-server="{{socks5://hostname:66}}" {{example.com}}`
+
+- Open with a custom profile directory:
+
+`chromium --user-data-dir={{path/to/directory}}`
+
+- Open without CORS validation, useful to test an API:
```
Comment (mfrw) [inline pages/common/chromium.md:30]: How about we use the format like we used for `Open in app mode`
```suggestion
- Open without CORS validation (useful to test an API):
```

**Review by mfrw — APPROVED** [review]
LGTM.
Very minor cosmetic nit.

File: pages/common/chromium.md [hunk 5]
```diff
@@ -1,15 +1,11 @@
...
+- Open a specific URL or file:
 
-`chromium {{path/to/file.html}}`
-
-- Open a URL:
-
-`chromium {{example.com}}`
+`chromium {{https://example.com|path/to/file.html}}`
 
 - Open in incognito mode:
```
Comment (EmilySeville7cfg) [inline pages/common/chromium.md:10]:
```suggestion
- Open in an incognito mode:
```

File: pages/common/chromium.md [hunk 6]
```diff
@@ -21,8 +17,20 @@
 
 - Open in app mode (without toolbars, URL bar, buttons, etc.):
```
Comment (EmilySeville7cfg) [inline pages/common/chromium.md:18]:
```suggestion
- Open in an app mode (without toolbars, URL bar, buttons, etc.):
```

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages/common/chromium.md (hunk 5 above)
Comment (navarroaxel) [reply pages/common/chromium.md:10]: not sure about this :thinking: I think the original description has a better grammar.

Comment (owenvoke) [reply pages/common/chromium.md:10]: I think the original here makes more sense. 👍🏻

**Pull request merged**